### ISSUE TITLE
[Issue] items not stacking after using removeItemFromPlayer()

### ISSUE THREAD

**Issue by mrh0** [Seniorendi/AdvancedPeripherals#93]
**Describe**

Items retrieved using removeItemFromPlayer() don't stack with items of the same type (items that should stack) that haven't passed through removeItemFromPlayer().

I bet it has something to do with nbt data.

**Reproduce**

1. call removeItemFromPlayer() using any item and amount into a chest
2. retrieve the items from the chest
3. attempt to stack these items with another stack of the same type.

**Screenshots/Videos**

https://i.gyazo.com/17ff088ba75110f01d4f6e644d1a4c6a.mp4

**Versions:**

- Forge version: 36.1.0
- AdvancedPeripherals version: 0.6.5b

**Comment by RGFTheCoder** [comment]
This seems to be because the [Function Definition](https://github.com/Seniorendi/AdvancedPeripherals/blob/master/src/main/java/de/srendi/advancedperipherals/common/addons/computercraft/peripheral/InventoryManagerPeripheral.java#L104) doesn't contain logic for copying NBT. It gets the base item from the registry and makes a stack of `count` size.

```java
Item item1 = ItemUtil.getRegistryEntry(item.get(), ForgeRegistries.ITEMS);    
stack = new ItemStack(item1, count);
```

**Comment by Seniorendi** [comment]
> This seems to be because the Function Definition doesn't contain logic for copying NBT. It gets the base item from the registry and makes a stack of count size.

Nop, this is just the code where we define the item which the player wants to move. 
We move the item at [this](https://github.com/Seniorendi/AdvancedPeripherals/blob/master/src/main/java/de/srendi/advancedperipherals/common/addons/computercraft/peripheral/InventoryManagerPeripheral.java#L129-L130) line of code. We go through every item in the chest/player inventory and check if the item is equal to the item of the item parameter.

**Comment by Seniorendi** [comment]
Mh, I can't reproduce this issue. I add 32 stone to the chest, and then I add the 32 stone to my inventory. I can always stack the items.

https://user-images.githubusercontent.com/67484093/121076424-b2b3f900-c7d6-11eb-9833-0a4254dc617e.mp4

**Comment by SirEndii** [comment]
Fixed in 0.7a
